content/docs/reference/pkg/azure/_index.md

```diff
@@ -101,3 +101,13 @@ A Pulumi package for creating and managing Microsoft Azure cloud resources.
     <li><a href="provider"><span class="symbol resource"></span>Provider</a></li>
 </ul>
 
+<h3>Package Details</h3>
+<dl class="package-details">
+	<dt>Repository</dt>
+	<dd>https://github.com/pulumi/pulumi-azure</dd>
+	<dt>License</dt>
+	<dd>Apache-2.0</dd>
+	<dt>Notes</dt>
+	<dd></dd>
```

Comment:
It seems like there is some issue with generating the package details in the doc. I'll take a look tomorrow.

Reply:
My suspicion is that it is likely because the pulumi-terraform-bridge dep might need to be updated in the provider repos since the change that adds it to the schema was only merged today.

Reply:
Yes - we'll need to update the providers to pull in the latest `pulumi-terraform-bridge` (and `pulumi`).  Alternatively we could suppress this `Notes` row if the data for it is not present in the schema for now.

Reply:
Opened https://github.com/pulumi/pulumi/pull/4275 for the latter to unblock things here while we wait on getting all the providers updated.